guides/common/modules/proc_creating-a-local-source-for-a-custom-file-type-repository.adoc

```diff
@@ -9,27 +9,26 @@ To create a file type repository in a directory on a remote server, see xref:Cre
 
 .Procedure
 ifdef::satellite[]
-. Ensure the Utils repository is enabled.
+. On your {ProjectServer}, enable the required repositories:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # subscription-manager repos \
 --enable={RepoRHEL8AppStream} \
 --enable={RepoRHEL8BaseOS} \
---enable={RepoRHEL8ServerSatelliteUtils}
-----
-. Enable the satellite-utils module:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# dnf module enable satellite-utils
+--enable={RepoRHEL8ServerSatelliteServerProjectVersion}
 ----
```

Comment:
I think that this step is not needed for the local source because those repos have to be enabled on an existing Satellite Server from installation already.

Reply:
Users may need to install the pulp manifest package on machines other than their Satellite, so it's fine to keep.

Reply:
Ah, this is for local installation, in that case it should not be necessary. However they will need to use foreman-maintain to install the RPM since foreman-protector will block the installation.

Reply:
I answered my own last question, the remote docs indeed should just use `dnf`. 

Feel free to remove the repository enabling bits, you're right that they'll already be enabled for local use of pulp manifest.